timeless/views.py

```diff
@@ -189,14 +188,27 @@ def get_default_context(self):
 class SingleObjectMixin:
     """ Fetch model from database using id """
     model = None
+    object_url_lookup = "id"
+
+    def get_object(self):
+        """ Takes object based on id provided in URL """
+        object_id = self.kwargs.get(self.object_url_lookup)
+        return self.model.query.get(object_id)
+
+
+class SuccessRedirectMixin:
```

Comment:
@vryazanov This SuccessRedirectMixin was clearly not part of the task, why did you add it? What about some tests for it?

Reply:
@amihaiemil this logic already was in `CreateView` and it's needed to me in `DeleteView`, so that's why I moved it to mixin class.